lib/snaplist_bloc.dart

```diff
@@ -7,6 +7,7 @@ class SnapListBloc {
   final CardSizeProvider sizeProvider;
   final SeparatorSizeProvider separatorProvider;
   final double swipeVelocity;
+  final bool isVertical;
```

Comment:
We should use the `Axis` from Dart, as it has only `horizontal` and `vertical` options. That way we explicitly define that if `isVertical == false` it will not go in circles.

Reply:
The `isVertical` boolean is still useful, it just should be a property.

Reply:
In the end, the param should be called `axis`, and the boolean will exist as `bool get isVertical => axis == Axis.vertical`.